app/src/main/res/values-v24/styles.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Depends on how you created it - possibly there are different defaults for pre-Nougat subtitle layout_marginTop and titleMarginTop